hal/armv8m/nrf/91/console.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Console
+ *
+ * Copyright 2022 Phoenix Systems
+ * Authors: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <hal/hal.h>
+#include "nrf91.h"
+#include <board_config.h>
+
+static struct {
+	volatile u32 *base;
+	u32 txDma;
```

Comment:
Yeah, but then there would be a following warning, because of `volatile u32 *base`:
![Screenshot from 2022-10-27 17-49-42](https://user-images.githubusercontent.com/77304431/198338185-b523447d-6184-46cc-a63f-1178671bb219.png)